idunn/places/models/pj_find.py

```diff
@@ -0,0 +1,329 @@
+
+
+class Pages(BaseModel):
+    current_page: Optional[int] = Field(None, description="Current page")
+    page_count: Optional[int] = Field(None, description="Total number of pages")
+    listings_per_page: Optional[int] = Field(None, description="Listings per page")
+    prev_page_url: Optional[str] = Field(None, description="URL to the previous page")
+    current_page_url: Optional[str] = Field(None, description="URL to the current page")
+    next_page_url: Optional[str] = Field(None, description="URL to the next page")
+
+
+class PracticalInformations(BaseModel):
+    is_health_insurance_card_accepted: Optional[bool] = Field(
+        None, description="Indicates if health insurance card is accepted"
+    )
+
+
+class Results(BaseModel):
+    first_listing: Optional[int] = Field(None, description="First listing index")
+    last_listing: Optional[int] = Field(None, description="Last listing index")
+    total_listing: Optional[int] = Field(None, description="Total number of listings")
+    total_local_listing: Optional[int] = Field(
+        None,
+        description="Total number of local listings. If argument proximity is true, this attribute is not available",
+    )
+
+
+class Reviews(BaseModel):
+    total_reviews: Optional[int] = Field(
+        None, description="Number of reviews that have been written for this listing"
+    )
+    overall_review_rating: Optional[float] = Field(
+        None, description="Overall rating for the review"
+    )
+
+
+class FilterEnum(Enum):
+    ECO = "ECO"
+    WEB_REVIEW = "WEB_REVIEW"
+    PRIVATE_ADVERTIZERS = "PRIVATE_ADVERTIZERS"
+    PRIVATE_OOREKA_PUSH = "PRIVATE_OOREKA_PUSH"
+    PRIVATE_OOREKA_LR = "PRIVATE_OOREKA_LR"
+    ECO_RESPONSABILITE = "ECO_RESPONSABILITE"
+    PAS_DE_LEURRE = "PAS_DE_LEURRE"
+    SANS_BOOSTER_GARANTI = "SANS_BOOSTER_GARANTI"
+    B2B = "B2B"
+
+
+class Search(BaseModel):
+    what: Optional[str] = Field(None, description="Search term text used in query")
+    where: Optional[str] = Field(None, description="The geographic location used in query")
+    latitude: Optional[float] = Field(
+        None, description="Latitude used in query (only if coordinates used in query)"
+    )
+    longitude: Optional[float] = Field(
+        None, description="Longitude used in query (only if coordinates used in query)"
+    )
+    proximity: Optional[bool] = Field(None, description="Proximity flag enabled in query")
+    sort: Optional[str] = Field(None, description="Sorting value set in query, if any")
+    filter: Optional[List[FilterEnum]] = Field(
+        None, description="Filtering value set in query, if any"
+    )
+    request_id: Optional[str] = Field(None, description="Request id (for stats purposes)")
+
+
+class Type(Enum):
+    RESERVER = "RESERVER"
+    RESERVER_INTERNE = "RESERVER_INTERNE"
+    COMMANDER = "COMMANDER"
+    COMMANDER_CHRONO = "COMMANDER_CHRONO"
+    PRENDRE_RDV_EXTERNE = "PRENDRE_RDV_EXTERNE"
+    PRENDRE_RDV_INTERNE = "PRENDRE_RDV_INTERNE"
+
```

Comment:
Side note: should we remove/comment these unused models ?  
Any breaking change in the API could break our app with no reason.

Reply:
At least it would be easy to update it by just re-downloading their openapi model :/

I'm not particularly convinced by both approach, neither "just not using any model at all". I'll remove unused parts as it feels indeed more robust.

Reply:
I removed unnecessary parts. This means that we are not afraid to stick to generated model so I took benefit of it by replacing all `Optional[List[...]]` by list fields which default to `[]` as this is slightly easier to work with :+1: